auth_user_insensitive/models/res_users.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015-2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResUsers(models.Model):
+
+    _inherit = 'res.users'
+
+    login = fields.Char(
+        'Login',
+        size=64,
```

Comment:
Remove, [no size limit in core](https://github.com/odoo/odoo/blob/9e159ef2048574d179a9afb2226397e962aa5725/odoo/addons/base/res/res_users.py#L193)